master/static/srv/proto_get_trial_workloads.sql

```diff
@@ -58,7 +64,7 @@ page_info AS (
 )
 SELECT (
   SELECT jsonb_agg(w) FROM (SELECT validation, training, checkpoint FROM workloads
-    ORDER BY total_batches %s, end_time %s
+    ORDER BY %s %s NULLS LAST, total_batches %s, end_time %s
```

Comment:
i think maybe there could be an extra `NULLS LAST` in here somewhere? sorting by loss asc results in nulls first

<img width="785" alt="image" src="https://user-images.githubusercontent.com/28383080/177257462-cfd76ecc-0627-4455-b715-0eceba0155e7.png">

Reply:
I don't have access to the same experiment, but as validation_loss would be desc here, I'm thinking that you're sorting by another column? And that column variable exists on both training and validation? I was thinking that it was going OK to sort these together, but I forgot how likely it is that only one of the columns would be displayed in the UI. So I probably want to revisit this and make it possible to treat these differently.